yt_dlp/extractor/zingmp3.py

```diff
@@ -424,3 +504,134 @@ def _real_extract(self, url):
             'sections', lambda _, v: v['sectionId'] == 'hub', 'items', ...)))
         return self.playlist_result(
             entries, song_id, hub_detail.get('title'), hub_detail.get('description'))
+
+
+class ZingMp3LiveRadioIE(ZingMp3BaseIE):
+    IE_NAME = 'zingmp3:liveradio'
+    _VALID_URL = r'https?://(?:mp3\.zing|zingmp3)\.vn/(?P<type>(?:liveradio))/(?P<id>\w+)(?:\.html|\?)'
+    _TESTS = [{
+        'url': 'https://zingmp3.vn/liveradio/IWZ979UB.html',
+        'info_dict': {
+            'id': 'IWZ979UB',
+            'title': r're:^V\-POP',
+            'description': 'md5:aa857f8a91dc9ce69e862a809e4bdc10',
+            'protocol': 'm3u8_native',
+            'ext': 'mp4',
+            'view_count': int,
+            'thumbnail': r're:^https?://.*\.jpg',
+            'like_count': int,
+        },
+        'params': {
+            'skip_download': True,
+        },
+    }, {
+        'url': 'https://zingmp3.vn/liveradio/IWZ97CWB.html',
+        'info_dict': {
+            'id': 'IWZ97CWB',
+            'title': r're:^Live\s247',
+            'description': 'md5:d41d8cd98f00b204e9800998ecf8427e',
+            'protocol': 'm3u8_native',
+            'ext': 'm4a',
+            'view_count': int,
+            'thumbnail': r're:^https?://.*\.jpg',
+            'like_count': int,
+        },
+        'params': {
+            'skip_download': True,
+        },
+    }]
+
+    def _real_extract(self, url):
+        url_type, live_radio_id = self._match_valid_url(url).group('type', 'id')
+        info = self._call_api(url_type, {'id': live_radio_id})
+        manifest_url = info.get('streaming')
+        if not manifest_url:
+            raise ExtractorError('This radio is offline.', expected=True)
+        fmts, subtitles = self._extract_m3u8_formats_and_subtitles(manifest_url, live_radio_id, fatal=False)
+        return {
+            'id': live_radio_id,
+            'title': traverse_obj(info, 'title'),
+            'thumbnail': traverse_obj(info, 'thumbnail', 'thumbnailM', 'thumbnailV', 'thumbnailH'),
+            'formats': fmts,
+            'view_count': traverse_obj(info, 'activeUsers'),
+            'like_count': traverse_obj(info, 'totalReaction'),
+            'description': traverse_obj(info, 'description'),
+            'subtitles': subtitles,
+        }
+
+
+class ZingMp3PostCastEpisodeIE(ZingMp3BaseIE):
```

Comment:
Is this (and other instances) supposed to be Podcast?

Reply:
@seproDev  Yes

Reply:
Can you update your code accordingly?

Reply:
I cant understand your question. Could you give more details?

Reply:
Your classes are currently named `PostCast`, I think this is supposed to be `Podcast`.
If this was a mistake, can you edit your PR so the classes have the correct name?

Reply:
`ZingMp3PostCastNewIE` with support for `postcast-new` type.

In `podcast-new` will have a list of podcast. And `ZingMp3PostCastEpisodeIE` will support for these new podcasts

Reply:
Your class is called ZingMp3Po**st**CastEpisodeIE instead of ZingMp3Po**d**castEpisodeIE. I was suggesting that this was a typo, since you are downloading podcasts not postcasts.

Reply:
Yes, its typo issue. I updated